src/components/sandbox/sandbox.jsx

```diff
@@ -31,6 +31,13 @@ const SandboxTitle = styled.h1`
     font-family: sans-serif;
     padding: 16px 0;
     word-break: break-word;
+
+    @media screen and (max-width: 375px) {
+        flex: 1 1 auto;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
when we have ellipsis, should we add title as html prop for tag?

Reply:
What do you mean by html prop? DangerousHtml?

Reply:
Added title attribute to header